src/utilities.js

```diff
@@ -107,13 +127,15 @@ export function getResults(searchType, obj) {
     },
   };
 
+  // Create an array of keywords to query
+  let comparisonItem = formatKeywords(obj);
   const options = {
     method: 'GET',
     host: 'trends.google.com',
     path: '/trends/api/explore',
     qs: {
       hl: obj.hl,
-      req: JSON.stringify({comparisonItem: [obj], cat: 0}),
+      req: JSON.stringify({comparisonItem, cat: 0}),
```

Comment:
And then here:
`re: JSON.stringify({comparisonItem: formatKeywords(obj), cat: 0}),`